app/src/main/java/com/framgia/fsalon/screen/customerinfo/user/UserFragment.java

```diff
@@ -78,4 +82,18 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
                 break;
         }
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        if (requestCode == 1) {
+            if (resultCode == RESULT) {
```

Comment:
WARNING: [CollapsibleIfStatements] These nested if statements could be combined
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/basic.html#CollapsibleIfStatements